scripts/userstyles.yml

```diff
@@ -176,14 +179,16 @@ userstyles:
   canvas-lms:
     name: Canvas LMS
     categories: [education, productivity]
+    icon: canvas
+    color: red
     readme:
       app-link: "https://www.instructure.com/canvas"
       current-maintainers: [*thememesniper]
   chatgpt:
     name: ChatGPT
     categories: [artificial_intelligence, productivity]
     icon: openai
-    color: green
+    color: mauve
```

Comment:
Why the change from green to mauve? Green seems to be more in line with the branding.

Reply:
![image](https://github.com/catppuccin/userstyles/assets/58985301/26f0e2f4-cc5a-4d69-8544-de56c5582ce5)

Reply:
This is ChatGPT, a product of OpenAI, rather than the company itself though.